scripts/new_project.kts

```diff
@@ -6,22 +6,32 @@ object NewProject {
     private const val DOT_SEPARATOR = "."
     private const val KEY_APP_NAME = "app-name"
     private const val KEY_PACKAGE_NAME = "package-name"
+    private const val MINUS_SEPARATOR = "-"
+    private const val SPACE_SEPARATOR = " "
     private const val TEMPLATE_APP_NAME = "Coroutine Template"
     private const val TEMPLATE_APPLICATION_CLASS_NAME = "CoroutineTemplateApplication"
     private const val TEMPLATE_FOLDER_NAME = "CoroutineTemplate"
     private const val TEMPLATE_PACKAGE_NAME = "co.nimblehq.coroutine"
 
-    private const val PATTERN_APP = "^[A-Z][a-zA-Z0-9\\s]*$"
+    private const val PATTERN_APP = "^([A-Z][a-zA-Z0-9\\s]*)|([a-z][a-z0-9-]*)$"
     private const val PATTERN_PACKAGE = "^[a-z]+(\\.[a-z][a-z0-9]*)+$"
 
     private val modules = listOf("app", "data", "domain")
     private val fileSeparator = File.separator
 
-    private var appName = ""
+    private var appName: String = ""
+        set(value) {
+            field = if (value.contains(MINUS_SEPARATOR)) {
+                value.replace(MINUS_SEPARATOR, SPACE_SEPARATOR).uppercaseEveryFirstWord()
+            } else {
+                value
```

Comment:
Should we also call `uppercaseEveryFirstWord()` here too? 😄 Just wondering why if I enter `todo-app` it becomes `Todo App`, but if I enter `Todo app` it doesn't become `Todo App`? 🤔

Reply:
@ryan-conway Nice catch 🚀 but since it wasn't in the AC, should I update it inside this PR? cc @Tuubz

Reply:
Nice, sounds good to me 🚀